lib/lcommon.pp

```diff
@@ -461,23 +461,28 @@ function IsPipeError(const anError: Integer): Boolean; inline;
 
 function TZSeconds: Integer; inline;
 begin
-  Result := unixutil.TZSeconds;
+{$IF FPC_FULLVERSION >= 33100}
+  Result := TZInfo.Seconds;
+{$ENDIF}
```

Comment:
could you make this info an else? otherwise we get a warning here since the compiler cannot infer that it's a full set

Reply:
@almindor  Ok I will look into it

Reply:
@almindor  It looks like original author use wrong `FPC_FULLVERSION`. For Free Pascal 3.3.1 or newer value should be `{$IF FPC_FULLVERSION >= 30301}` while for Free Pascal  3.3.0 or older `{$IF FPC_FULLVERSION <= 30300}`.

[Read FPC_FULLVERSION documentation here](https://www.freepascal.org/docs-html/prog/progap7.html).